src/runtime/core/CMakeLists.txt

```diff
@@ -150,7 +150,7 @@ endif()
 INSTALL(FILES
         ${CMAKE_CURRENT_SOURCE_DIR}/../../../include/atmi.h
         ${CMAKE_CURRENT_SOURCE_DIR}/../../../include/atmi_runtime.h
-        DESTINATION "include"
+        DESTINATION "include/atmi"
```

Comment:
How about the install path for the `.so` libraries? Do they remain as-is with `DESTINATION lib` and `lib-debug`? Or will they go to `lib/atmi` and `lib-debug/atmi`?

Reply:
The libraries will remain in Destination lib and lib-debug